packages/common/src/visualizations/SqlRunnerResultsTransformer.ts

```diff
@@ -136,7 +159,11 @@ type SqlRunnerResultsTransformerDeps = {
 };
 
 export class SqlRunnerResultsTransformer
-    implements ResultsTransformerBase<SqlTransformBarChartConfig>
+    implements
+        ResultsTransformerBase<
+            SqlTransformBarChartConfig,
+            SqlTransformPieChartConfig
```

Comment:
As discussed, I put the pie chart getters on this class, so it takes both generics. We could make sub classes if we don't like this. Or make the data types more shared? For example, instead of barChartXLayoutOptions, could it be dimensionOptions?